onnx/common/ir.h

```diff
@@ -39,6 +39,16 @@
 
 namespace ONNX_NAMESPACE {
 
+namespace { // internal/private API
+
+std::string toVarName(size_t i) {
+  std::ostringstream oss;
+  oss << "_v_" << i;
```

Comment:
nit: Would it make sense to just use a formatting string for this?

Reply:
Which formatting string / API are you referring to?

Reply:
Looks like std::format isn’t until c++ 20. NVM 😅

Reply:
right